### DISCUSSION TITLE
Google Groups Closed?

### DISCUSSION THREAD

**Discussion by daBee** [jeremyevans/sequel#2156] [General]
Just viewed the Google Groups and there's Banned Content in there somehow.  What's up with that?  Are discussions now in here?

**Comment by jeremyevans** [comment]
Google banned the group.  Discussions are now here.  You can see a partial archive of the Google Group at https://www.mail-archive.com/sequel-talk@googlegroups.com/

**Comment by AlexWayfer** [comment]
You could check the existing discussion with additional information: https://github.com/jeremyevans/sequel/discussions/2145
